### PR TITLE
added in change from A/B test homepage-and-nav-cta-button-copy experi…

### PR DESCRIPTION
**Pull request by thtmnisamnstr**
…ment. Fixed a few nits that came up in a website review call.

### CODE REVIEW

**Comment by thtmnisamnstr** [comment]
Preview URL: https://645c1b47a8bbbe0b1c7edf0d--ephemeral-narwhal-3ac3b2.netlify.app

**Comment by thtmnisamnstr** [comment]
Preview URL: https://645c1d10d2d1ff0528c64613--ephemeral-narwhal-3ac3b2.netlify.app

File: website/_includes/head.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
   {% if jekyll.environment == "production" %}
   {%- include segment.html -%}
-  {%- include growthbook.html -%}
+  <!-- {%- include growthbook.html -%} -->
```
Comment (vladaionescu) [inline website/_includes/head.html:31]: Intended change?

**Review by vladaionescu — APPROVED** [review]

File: website/_includes/head.html (hunk 1 above)
Comment (thtmnisamnstr) [reply website/_includes/head.html:31]: Yep. I remove the growthbook script when it isn't in use.

**Pull request merged**